libp2p/protocols/mix/entry_connection.nim

```diff
@@ -5,14 +5,27 @@ import ../../utils/sequninit
 import ./mix_protocol
 from fragmentation import DataSize
 
+const DefaultSurbs = uint8(4) # Default number of SURBs to send
+
 type MixDialer* = proc(
   msg: seq[byte], codec: string, destination: MixDestination
 ): Future[void] {.async: (raises: [CancelledError, LPStreamError], raw: true).}
 
+## Holds configuration parameters used in mix protocol operations.  
+type MixParameters* = object
+  expectReply*: Opt[bool]
+  numSurbs*: Opt[uint8]
+
 type MixEntryConnection* = ref object of Connection
   destination: MixDestination
   codec: string
   mixDialer: MixDialer
+  params: Opt[MixParameters]
```

Comment:
if all values of `MixParameters` are optional do we really need to have `params` to be optional as well?